modules/ocf/manifests/packages.pp

```diff
@@ -110,6 +110,7 @@
     'python3-pip',
     'python3-requests',
     'python3-tabulate',
+    'qrencode',
```

Comment:
Do you really need this on *all* hosts at the OCF? Even hosts like LDAP, mysql, etc?

This probably belongs either in ocf_desktop packages or extrapackages (depending on if it's only useful with a GUI or can be useful on a headless server).